README.md

```diff
@@ -38,116 +38,53 @@ Here are a few key pages you might be interested in:
 
 ## Quickstart
 
-TigerBeetle is easy to run with or without Docker, depending on your
-preference. First, we'll cover running the [Single
-Binary](#single-binary). And below that is how to run [with
-Docker](#with-docker).
+First, grab a copy of TigerBeetle.
 
-### Single Binary
+On macOS/Linux:
 
-Install TigerBeetle by grabbing the latest release from
-GitHub.
-
-x86_64 and aarch64 builds are available for macOS and Linux. Only
-x86_64 builds are available for Windows.
-
-For example:
-
-```bash
-$ curl -LO https://github.com/tigerbeetledb/tigerbeetle/releases/download/2023-03-27-weekly/tigerbeetle-x86_64-linux-2023-03-27-weekly.zip
-$ unzip tigerbeetle-x86_64-linux-2023-03-27-weekly.zip
-$ sudo cp tigerbeetle /usr/local/bin/tigerbeetle # On Windows, add $(pwd) to $env:PATH instead.
-$ tigerbeetle version --verbose | head -n6
-TigerBeetle version experimental
-
-git_commit="55c8fdf1f52c7a174d1bc9d9785cf4e327cae182"
-
-build.mode=Mode.ReleaseSafe
-build.zig_version=0.9.1
+```console
+git clone https://github.com/tigerbeetledb/tigerbeetle; ./tigerbeetle/scripts/bootstrap.sh
 ```
 
-NOTE: This example version is not kept up-to-date. So always check the
-[releases](https://github.com/tigerbeetledb/tigerbeetle/releases) page
-for the latest version. You can also find debug builds for each
-arch/OS combo on the release page as well.
-
-#### Building from source
-
-Or to build from source, clone the repo, checkout a release, and run
-the install script.
+On Windows:
 
-You will need POSIX userland, curl or wget, tar, and xz.
-
-```bash
-$ git clone https://github.com/tigerbeetledb/tigerbeetle.git
-$ cd tigerbeetle
-$ git checkout 2022-11-16-weekly # Or latest tag
-$ scripts/install.sh
+```console
+git clone https://github.com/tigerbeetledb/tigerbeetle; ./tigerbeetle/scripts/bootstrap.ps1
 ```
 
-Don't worry, this will only make changes within the `tigerbeetle`
-directory. No global changes. The result will place the compiled
-`tigerbeetle` binary into the current directory.
+Want to build from source? Add `-build` to the bootstrap script arguments:
```

Comment:
`-build`  as opposed to `--from-source` because Windows deals with `--` flags in a weird way.